ajax/libs/minireset.css/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "minireset.css",
+  "version": "0.0.2",
+  "description": "A tiny modern CSS reset",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/jgthms/minireset.css.git"
+  },
+  "keywords": [
+    "css",
+    "sass",
+    "reset",
+    "mini"
+  ],
+  "author": "Jeremy Thomas <bbxdesign@gmail.com> (http://jgthms.com)",
+  "license": "MIT",
+  "npmName": "minireset.css",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "docs"
```

Comment:
This could be invalid, should point to a file not a directory ... we'll need to update the test,